mafiasi/gprot/admin.py

```diff
@@ -1,6 +1,6 @@
 from django.contrib import admin
 
-from mafiasi.gprot.models import Attachment, GProt, Notification, Reminder
+from mafiasi.gprot.models import Attachment, GProt, Notification, Reminder, Label
```

Comment:
This breaks the alphabeticla order of the import items, let your IDE reformat them :)

Reply:
they are used in the django admin interface